docker-compose.circleci.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   prisoner-content-hub-backend-db:
-    image: mariadb
+    image: mariadb:10.4
```

Comment:
Should we be using a more recent version of mariadb?

Reply:
That would involve updating the prod version. Which we could do, but may result in some downtime.